### PR TITLE
HTML script element support for type=importmap

### LINKED ISSUE #17701: Missing data for Import Maps
### What type of issue is this?

Missing compatibility data

### What information was incorrect, unhelpful, or incomplete?

Import Maps were, per https://chromestatus.com/feature/5315286962012160, shipped in Chrome 89.

However, there are no links to https://wicg.github.io/import-maps/ anywhere within BCD.

### What browsers does this problem apply to, if applicable?

_No response_

### What did you expect to see?

_No response_

### Did you test this? If so, how?

_No response_

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN URL

_No response_

### MDN metadata

_No response_

### PR DESCRIPTION
**Pull request by hamishwillee**
FF108 adds support for "import maps", a mechanisms for providing a mapping between a module name and its location when you're importing a module. The feature is implemented using an HTML script element that has `type="importmap"`, where the element contains the import map as a JSON definition.

Compat:
- Firefox 102 - 108 implemented behind a preference: https://bugzilla.mozilla.org/show_bug.cgi?id=1688879
- FF108 ships: https://bugzilla.mozilla.org/show_bug.cgi?id=1795647
- Chrome 89: https://chromestatus.com/feature/5315286962012160
- Safari - not implemented (see chrome signal, and canIuse).

Fixes #17701

Other docs work can be tracked in: https://github.com/mdn/content/issues/16150

FYI @queengooborg

### CODE REVIEW

File: html/elements/script.json [hunk 1]
```diff
@@ -463,6 +463,51 @@
...
+                    "version_removed": "108",
+                    "flags": [
+                      {
+                        "type": "preference",
+                        "name": "dom.importMaps.enabled",
+                        "value_to_set": "true"
+                      }
+                    ]
+                  }
+                ],
```
Comment (queengooborg) [inline html/elements/script.json:489]:
```suggestion
                "firefox": {
                  "version_added": "108"
                },
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
LGTM, but let's remove the flag data -- since Firefox 108 is coming out soon, we're going to remove it soon anyways!

File: html/elements/script.json [hunk 2]
```diff
@@ -463,6 +463,51 @@
               "deprecated": false
             }
           },
+          "importmap": {
+            "__compat": {
+              "support": {
```
Comment (hamishwillee) [inline html/elements/script.json:468]:
```suggestion
            "__compat": {
              "spec_url": "https://html.spec.whatwg.org/multipage/webappapis.html#import-map",
              "support": {
```

**Comment by hamishwillee** [comment]
Thanks @queengooborg -ready for re-review

**Review by queengooborg — APPROVED** [review]

**Pull request merged**